ext/php7/handlers_pcntl.c

```diff
@@ -14,11 +17,13 @@ ZEND_FUNCTION(ddtrace_pcntl_fork) {
     dd_pcntl_fork_handler(INTERNAL_FUNCTION_PARAM_PASSTHRU);
     if (Z_LVAL_P(return_value) == 0) {
         // CHILD PROCESS
-        // Until we full support pcntl tracing:
-        //   - kill the BGS
-        //   - disable further tracing on the forked process (also ensures all spans are dropped)
         ddtrace_coms_kill_background_sender();
-        ddtrace_disable_tracing_in_current_request();
+        ddtrace_init_span_stacks();
```

Comment:
I think this needs a `ddtrace_free_span_stacks();`  before, to avoid leaking existing spans.